### DISCUSSION TITLE
[Q] Can Xposed modules not in lsposed repo be used?

### DISCUSSION THREAD

**Discussion by DarkenLX** [LSPosed/LSPosed#1259] [Q&A] [answered]
Hi basically my question is if say a Xposed module from say Edxposed for example has not gotten added To the lsposed repos yet is it possible to used a module not specifically built for lsposed? What would happen? And before I get asked I'm just looking for one of the decent Pokemon go modules mainly the joystick ones..

**Comment by JasonKhew96** [comment, marked as answer]
@DarkenLX 
> Hi basically my question is if say a Xposed module from say Edxposed for example has not gotten added To the lsposed repos yet is it possible to used a module not specifically built for lsposed?

Yes, but you need to figure out the module scope yourself

**Reply by oncebeloved** [reply]
作用域要如何弄清呢？
